README.md

```diff
@@ -2,30 +2,29 @@
 
 [![Version](https://vsmarketplacebadge.apphb.com/version/msazurermtools.azurerm-vscode-tools.svg)](https://marketplace.visualstudio.com/items?itemName=msazurermtools.azurerm-vscode-tools) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/msazurermtools.azurerm-vscode-tools.svg)](https://marketplace.visualstudio.com/items?itemName=msazurermtools.azurerm-vscode-tools) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/vscode-azurearmtools)](https://dev.azure.com/ms-azuretools/AzCode/_build/latest?definitionId=10)
 
-This extension provides language support for Azure Resource Manager deployment templates and template language expressions.
+The Azure Resource Manager (ARM) Tools for Visual Studio Code provides language support, resource snippets, and resource auto-completion to help you create and validate Azure Resource Manager templates.
 
-## Features
+![ARM Tools creating web app](./images/arm-tools.gif)
 
-- Provides a language server that understands Azure Resource Manager deployment template files
+## Language Server
+
+- Azure Resource Manager template language server for providing ARM template language completion, validation, and error guidance.
+- Resources are validated against Azure schemas which provides validation against resource type and apiVersion properties.
```

Comment:
how about "based on resource type and apiVersion"?

Reply:
Like this?

Resources are validated against Azure schemas which provides validation based on resource type and apiVersion.

Reply:
Yes, but grammar nit: Ambiguous what "which" refers to, making it unclear whether it should be "provide" or "provides".  How about a comma after "schemas"?